app/views/content/is-teaching-right-for-me/teach-disabled-pupils-and-pupils-with-special-educational-needs.md

```diff
@@ -34,6 +34,12 @@ As you train and teach you'll get the skills you need to work with disabled pupi
 
 This will enable you to work in a range of settings throughout your career.
 
+> I specifically wanted to work with children identified with Special Educational Needs and Disabilities (SEND). I was fortunate to find a teacher training course provider that was able to allow me to practice in a mainstream school that also hosted a hearing impairment unit. And another school that offered support for children with a variety of learning needs.
+>
+> They were great at teaching me and letting me practice my classroom management skills early on. I was able to research and understand ways to differentiate resources for children who were on the Autistic Spectrum. And I was able to create and implement a scheme of work to reflect on and evaluate for later use once qualified.
+>
+>_Jane Wilkinson, former teacher and teacher training adviser._
+
```

Comment:
@gemmadallmandfe  I just noticed that we refer to Jane Wilkinson as a former teacher and training advisor. Do we need to amend this to say training advisor with years of teaching experience?

Reply:
I think it is probably fine in this case - it would be very clunky otherwise?